types/module/module.go

```diff
@@ -425,21 +435,26 @@ func (m *Manager) checkModulesExists(moduleName []string) error {
 
 // assertNoForgottenModules checks that we didn't forget any modules in the
 // SetOrder* functions.
```

Comment:
Can you add godoc on what pass is for?

Reply:
I think the documents under `docs/docs/basic` should be fine as is.
I've added the description about `pass` in the corresponding godoc.